src/components/ui/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import { Meta } from "@storybook/react";
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "./Accordion";
+import { Card } from "@ui/Card";
+export default {
+  component: Accordion,
+  title: "Components/Accordion",
```

Comment:
What's the difference between a component and a UI Element? Should this live in the UI Elements with the others?

Reply:
good catch, moved to uielements in f2d90dd